src/gui/raster/qgsrasterlayerproperties.cpp

```diff
@@ -269,6 +271,18 @@ QgsRasterLayerProperties::QgsRasterLayerProperties( QgsMapLayer *lyr, QgsMapCanv
   layout->addWidget( mMetadataWidget );
   metadataFrame->setLayout( layout );
 
+  // Temporal options
+
+  if ( mRasterLayer->temporalProperties()->isActive() ||
+       mRasterLayer->dataProvider()->temporalCapabilities()->isActive() )
```

Comment:
Is there no risk that either `dataProvider()` or `temporalCapabilities()` are `nullptr`?